templates/erp/includes/fields.html

```diff
@@ -1,41 +1,31 @@
 {% load a4a %}
 {% load static %}
 {% load i18n %}
-{% if fields %}
-    <ul class="p-0 m-0">
-        {% for field in fields %}
-            {% render_field field.value as field_value %}
-            {% if field_value %}
```

Comment:
Removed, the logic was added in the python to avoid:

- Building all the dicts
- Removing the one with all empty field (python)
- Hiding those without values (template)

All the work is now done in python to avoid back and forth to understand why a value is displayed or not